benefits/core/admin/transit.py

```diff
@@ -1,48 +1,74 @@
+from django.conf import settings
 from django.contrib import admin
 
 from benefits.core import models
+from .users import is_staff_member_or_superuser
 
 
 @admin.register(models.TransitAgency)
-class TransitAgencyAdmin(admin.ModelAdmin):  # pragma: no cover
+class TransitAgencyAdmin(admin.ModelAdmin):
     def get_exclude(self, request, obj=None):
+        fields = []
+
         if not request.user.is_superuser:
-            return [
-                "eligibility_api_private_key",
-                "eligibility_api_public_key",
-                "transit_processor_client_id",
-                "transit_processor_client_secret_name",
-                "transit_processor_audience",
-            ]
-        else:
-            return super().get_exclude(request, obj)
+            fields.extend(
+                [
+                    "eligibility_api_private_key",
+                    "eligibility_api_public_key",
+                    "sso_domain",
+                    "transit_processor_audience",
+                    "transit_processor_client_id",
+                    "transit_processor_client_secret_name",
+                ]
+            )
+
+        return fields or super().get_exclude(request, obj)
 
     def get_readonly_fields(self, request, obj=None):
+        fields = []
+
         if not request.user.is_superuser:
-            return [
-                "eligibility_api_id",
-                "transit_processor",
-                "index_template_override",
-                "eligibility_index_template_override",
-            ]
+            fields.extend(
+                [
+                    "index_template_override",
+                    "eligibility_index_template_override",
+                    "eligibility_api_id",
+                    "transit_processor",
+                ]
+            )
+
+        return fields or super().get_readonly_fields(request, obj)
+
+    def has_add_permission(self, request):
+        if settings.RUNTIME_ENVIRONMENT() != settings.RUNTIME_ENVS.PROD:
+            return True
+        elif request.user and is_staff_member_or_superuser(request.user):
+            return True
         else:
-            return super().get_readonly_fields(request, obj)
+            return False
 
 
 @admin.register(models.TransitProcessor)
-class TransitProcessorAdmin(admin.ModelAdmin):  # pragma: no cover
-    def get_exclude(self, request, obj=None):
-        if not request.user.is_superuser:
-            return []
-        else:
-            return super().get_exclude(request, obj)
+class TransitProcessorAdmin(admin.ModelAdmin):
```

Comment:
I was reviewing this model in the Admin UI and thought... a Cal-ITP Staff probably doesn't need to see the fields we currently have as read-only for them (`card_tokenize_*` and `api_base_url`). It's more visual noise that they cannot act on anyway.

I updated the spreadsheet, can we exclude these for all but `superuser`?

Reply:
Yep, that makes sense, I updated the permissions in 6bc5f536c583ae7d0e1e44fa5f1c97325778a6b6